packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -371,7 +392,9 @@ export async function runNpmInstall(
   try {
     await runNpmInstallSema.acquire();
     const { cliType, packageJsonPath, lockfileVersion } = await scanParentDirs(
-      destPath
+      destPath,
+      true,
+      spawnOpts?.env?.ENABLE_EXPERIMENTAL_COREPACK === '1'
```

Comment:
Maybe we should pass the whole `spawnOpts` instead?

Reply:
sure